ra/SAPHana

```diff
@@ -798,7 +798,7 @@ function saphana_init() {
     export SAPSYSTEMNAME=$SID
     HANA_CALL_TIMEOUT="${OCF_RESKEY_HANA_CALL_TIMEOUT:-60}"
     super_ocf_log debug "DBG: Used new method to get SID ($SID) and InstanceNr ($InstanceNr)"
-    sid=$(echo "$SID" | tr [:upper:] [:lower:])
+    sid="$SID"
```

Comment:
I prefer to do that more explicitly like: sid="${SID,,}"

Reply:
SID,, and SID^^ are only available since BASHv4 .. SLES11 and RHEL6 are shipped with BASHv3 ..